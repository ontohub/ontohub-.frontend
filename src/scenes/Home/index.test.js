import React from "react";
import { Home, Header } from ".";
import { ThemeProvider } from "emotion-theming";
import { render } from "enzyme";

describe("Home", () => {
  describe("Header", () => {
    it("matches the snapshot", () => {
      let wrapper = render(
        <ThemeProvider theme={{ sizes: { headerPadding: "1em" } }}>
          <Header />
        </ThemeProvider>
      );

      expect(wrapper).toMatchSnapshot();
    });
  });

  describe("Body", () => {
    it("matches the snapshot", () => {
      let wrapper = render(<Home />);

      expect(wrapper).toMatchSnapshot();
    });
  });
});
