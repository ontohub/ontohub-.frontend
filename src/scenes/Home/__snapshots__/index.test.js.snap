// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Body matches the snapshot 1`] = `
<div
  class="ui text container"
>
  <div
    class="ui tiny icon warning message"
  >
    <i
      aria-hidden="true"
      class="announcement icon"
    >
      
    </i>
    <div
      class="content"
    >
      <div
        class="header"
      >
        Development preview
      </div>
      You are currently viewing an early development version of the new Ontohub website. It does not yet include much functionality. To use the stable version of Ontohub, please visit 
      <a
        href="https://ontohub.org"
      >
        ontohub.org
      </a>
      .
    </div>
  </div>
</div>
`;

exports[`Home Header matches the snapshot 1`] = `
<div
  class="ui container sc-EHOje bwZcKZ"
>
  <h2
    class="ui inverted header"
  >
    Welcome to Ontohub
  </h2>
  <div
    class="ui large list"
    role="list"
  >
    <div
      class="item"
      role="listitem"
    >
      <i
        aria-hidden="true"
        class="fork icon"
      >
        
      </i>
      <div
        class="content"
      >
        <span
          class="sc-bdVaJa koclZm"
        >
          Open
        </span>
         — based on open source software
      </div>
    </div>
    <div
      class="item"
      role="listitem"
    >
      <i
        aria-hidden="true"
        class="code icon"
      >
        
      </i>
      <div
        class="content"
      >
        <span
          class="sc-bdVaJa koclZm"
        >
          Flexible
        </span>
         — supporting OWL, UML, FOL/TPTP, HOL/THF, and more
      </div>
    </div>
    <div
      class="item"
      role="listitem"
    >
      <i
        aria-hidden="true"
        class="sitemap icon"
      >
        
      </i>
      <div
        class="content"
      >
        <span
          class="sc-bdVaJa koclZm"
        >
          Distributed
        </span>
         — OMS alignments, mappings, networks, combinations using DOL
      </div>
    </div>
  </div>
</div>
`;
